mlir/test/Dialect/Vector/linearize.mlir

```diff
@@ -1,17 +1,25 @@
 // RUN: mlir-opt %s -split-input-file -test-vector-linearize | FileCheck %s
+// RUN: mlir-opt %s -split-input-file -test-vector-linearize=target-vector-bitwidth=128 | FileCheck %s
+// RUN: mlir-opt %s -split-input-file -test-vector-linearize=target-vector-bitwidth=12 | FileCheck %s --check-prefix=CHECK12
```

Comment:
It looks like you wanna test all ops are not linearized? I'm saying that because all the element types has bitwidth greater than 12. If so, maybe just test with `bitwidth=0` and make check prefix more specific.